example/src/DieHard.hs

```diff
@@ -177,6 +176,15 @@ ixfor _ EmptySmall   _ = pure EmptySmall
 ixfor _ SmallIntoBig _ = pure SmallIntoBig
 ixfor _ BigIntoSmall _ = pure BigIntoSmall
 
+instance IxFunctor  (Untyped Step) where
+  ifmap _ _ = undefined
```

Comment:
I know, I know... I'll try to fix this `IxFunctor *` and `IxFunctor1` stuff next.